### PR TITLE
For #85: added test for repomd.xml update

### PR DESCRIPTION
**Pull request by paulodamaso**
For #85:
-  added test for repomd.xml update
- left puzzle for implementing the update test for the rest of files

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#157](https://github.com/artipie/rpm-adapter/issues/157) is assigned to @vatavuk[/z](https://www.0crat.com/u/vatavuk), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/933a84b9-76bb-42b2-85f1-c66f9b827373); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/test/java/com/artipie/rpm/RpmITCase.java [hunk 1]
```diff
@@ -79,4 +82,42 @@ void generatesMetadata(@TempDir final Path tmp) throws Exception {
...
+        );
+        new Gzip(bundle).unpack(repo);
+        Files.delete(bundle);
+        final Storage storage = new FileStorage(repo, this.vertx.fileSystem());
+        new Rpm(storage, StandardNamingPolicy.SHA1, Digest.SHA256, true)
+            .batchUpdate(Key.ROOT)
+            .blockingAwait();
+        MatcherAssert.assertThat(
+            new String(
+                Flowable.fromPublisher(
```
Comment (Vatavuk) [inline src/test/java/com/artipie/rpm/RpmITCase.java:100]: @paulodamaso we can simplify to:
```java
new Concatenation(
   storage.value(new Key.From("repodata/repomd.xml")).get()
).single().blockingGet().array(),
Charset.defaultCharset()
```

**Comment by paulodamaso** [comment]
@Vatavuk Done, please take a look

**Comment by paulodamaso** [comment]
@g4s8 ping

**Comment by 0crat** [comment]
@vatavuk[/z](https://www.0crat.com/u/vatavuk) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @vatavuk[/z](https://www.0crat.com/u/vatavuk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@Vatavuk please make sure you will find at least 3 issues during next CR, thanks

**Comment by Vatavuk** [comment]
@sereshqua ok

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Pull request merged**